### PR TITLE
Update environment variables guide to use webpack CLI option `--env`

### PR DESCRIPTION
**Pull request by grisanu**
- Update environment variables <a href="https://webpack.js.org/guides/environment-variables/">guide</a> to use webpack CLI option `--env` instead of recommending `cross-env`
- Link section in webpack CLI <a href="https://webpack.js.org/api/cli/#environment-options">Environment Options</a> section to guide

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ The `--env` argument accepts various syntaxes:
 | webpack --env.prod --env.min  | `{ prod: true, min: true }` |
 | webpack --env.prod --env min  | `[{ prod: true }, "min"]`   |
 
+T> See this [environment variables](/guides/environment-variables) guide for more information.
```
Comment (TheLarkInn) [inline src/content/api/cli.md]: I know it's not a relevant change that you have made, but I noticed we were missing one more --env usage format! 

For example if you wanted to pass an "array" of info to the same property you can do so like this: 

`--env.foo=hello --env.foo=world`

And inside of the config would return the following for `env`

`{foo: ['hello', 'world']}`

Would you be willing to add this information in this PR?

Comment (grisanu) [reply src/content/api/cli.md]: :+1:

File: src/content/api/cli.md [hunk 2]
```diff
@@ -130,14 +130,17 @@ webpack --env.platform=web  # sets env.platform == "web"
...
+|----------------------------------------|-----------------------------|
+| webpack --env prod                     | `"prod"`                    |
+| webpack --env.prod                     | `{ prod: true }`            |
+| webpack --env.prod=1                   | `{ prod: 1 }`               |
+| webpack --env.prod=foo                 | `{ prod: "foo" }`           |
+| webpack --env.prod --env.min           | `{ prod: true, min: true }` |
+| webpack --env.prod --env min           | `[{ prod: true }, "min"]`   |
+| webpack --env.prod=foo --env.prod=bar  | `{prod: [ "foo", "bar" ]}`  |
+
+T> See this [environment variables](/guides/environment-variables) guide for more information on its usage.
```
Comment (ChrisChinchilla) [inline src/content/api/cli.md]: @grisanu Better to use 'See the' as there is only one environment guide.

File: src/content/guides/environment-variables.md [hunk 3]
```diff
@@ -3,35 +3,48 @@ title: Environment Variables
...
+  - title: The Fine Art of the webpack 3 Config
+    url: https://blog.flennik.com/the-fine-art-of-the-webpack-2-config-dc4d19d7f172#d60a
 ---
 
 To disambiguate in your `webpack.config.js` between [development](/guides/development) and [production builds](/guides/production), you may use environment variables.
 
-The standard approach in Node.js modules can be applied: Set an environment variable when running webpack and refer to the variables using Node's [`process.env`](https://nodejs.org/api/process.html#process_process_env). The variable `NODE_ENV` is commonly used as de-facto standard (see [here](https://dzone.com/articles/what-you-should-know-about-node-env)).
+The webpack command line [environment option](/api/cli/#environment-options), `--env` allows you to pass in as many environment variables as you would like. Environment variables will be made accesible in your `webpack.config.js`. For example, `--env.production` or `--env.NODE_ENV=local` (`NODE_ENV` is conventionally used to define the environment type, see [here](https://dzone.com/articles/what-you-should-know-about-node-env).)
```
Comment (ChrisChinchilla) [inline src/content/guides/environment-variables.md]: @grisanu You can probably remove 'would' from 'as you would like', it serves no purpose here.

File: src/content/guides/environment-variables.md [hunk 4]
```diff
@@ -3,35 +3,48 @@ title: Environment Variables
...
-+      compress: process.env.NODE_ENV === 'production'
-    })
-  ]
-};
+```bash
+webpack --env.NODE_ENV=local --env.production --progress
 ```
 
-Use the [`cross-env`](https://www.npmjs.com/package/cross-env) package to cross-platform-set environment variables:
+T> Setting up your `env` variable without assignment, `--env.prduction` sets `env.prduction` to `true` by default. There are also other various syntaxes that you can use. See the [webpack CLI](/api/cli/#environment-options) documentation for more information.
```
Comment (ChrisChinchilla) [inline src/content/guides/environment-variables.md]: @grisanu I would probably say 'the assignment' unless I'm missing something, and also do you mean `--env.production`?

You don't really need the 'various' really.

**Comment by TheLarkInn** [comment]
@ChrisChinchilla thank you for jumping in and helping with these kind of contributions. @grisanu if anything is unclear don't hesitate to chime in. Happy to advise and assist as well.

**Comment by grisanu** [comment]
Thank you for all of your suggestions! I've made changes and pushed. Please let me know what you think! @TheDutchCoder @ChrisChinchilla

**Comment by TheDutchCoder** [comment]
^^ @TheLarkInn I think ;)

**Comment by grisanu** [comment]
@skipjack Thank you! :+1:

**Pull request merged**